docker-compose.yml

```diff
@@ -36,8 +36,8 @@ services:
         target: /srv/ledgersmb
         source: ${PWD}
     environment:
-      LSMB_BASE_URL: http://lsmb-devel:5762
-      PSGI_BASE_URL: http://lsmb-devel:5762
+      LSMB_BASE_URL: http://ledgersmbdevdocker_lsmb_1:5762
+      PSGI_BASE_URL: http://ledgersmbdevdocker_lsmb_1:5762
```

Comment:
Hmm. my containers are named `ldmaster_lsmb_1` and `ld15_lsmb_1`.

Wondering if it makes sense to put the container name back into the equation. maybe based on the `${COMPOSE_PROJECT_NAME}` variable? (Which is what makes mine be prefixed with `ld<version>`.)